File: python/models/jurassic2.py
import boto3
import json

client = boto3.client(service_name="bedrock-runtime", region_name="us-east-1")

prompt = "Hello, how are you today?"
model_id = "ai21.j2-mid-v1"

request_body = json.dumps({"prompt": prompt})

response = client.invoke_model(
    modelId=model_id,
    body=request_body
)

result = json.loads(response.get("body").read())
message = result["completions"][0]["data"]["text"]

print(f"Response: {message}")

File: javascript/models/jurassic2.js
import {BedrockRuntimeClient, InvokeModelCommand} from "@aws-sdk/client-bedrock-runtime";

const client = new BedrockRuntimeClient({region: "us-east-1"});

const prompt = "Hello, how are you today?";
const modelId = "ai21.j2-mid-v1";

const body = JSON.stringify({prompt});

const response = await client.send(new InvokeModelCommand({
    body, modelId, contentType: "application/json",
}));

const result = JSON.parse(new TextDecoder().decode(response.body));
const message = result.completions[0].data.text;

console.log(`Response: ${message}`);
